yellowbrick/features/pca.py

```diff
@@ -144,25 +148,30 @@ class PCADecomposition(FeatureVisualizer):
     >>> visualizer.poof()
 
     """
-    def __init__(self, ax=None, scale=True, color=None, proj_dim=2,
-                 colormap=palettes.DEFAULT_SEQUENCE, **kwargs):
-        super(PCADecomposition, self).__init__(ax=ax, **kwargs)
-
+    def __init__(self, features=None, ax=None, scale=True, color=None,
+                 proj_dim=2, colormap=palettes.DEFAULT_SEQUENCE,
+                 proj_features=False, **kwargs):
+        super(PCADecomposition, self).__init__(ax=ax,
+                                               features=features,
+                                               **kwargs)
         # Data Parameters
         if proj_dim not in (2, 3):
             raise YellowbrickValueError("proj_dim object is not 2 or 3.")
-
         self.color = color
         self.scale = scale
         self.proj_dim = proj_dim
-        self.pca_transformer = Pipeline([('scale', StandardScaler(with_std=self.scale)),
-                                         ('pca', PCA(self.proj_dim, ))
-                                         ])
+        self.pca_transformer = Pipeline(
+            [('scale', StandardScaler(with_std=self.scale)),
+             ('pca', PCA(self.proj_dim, ))]
+        )
+        self.proj_features = proj_features
         # Visual Parameters
         self.colormap = colormap
 
     def fit(self, X, y=None, **kwargs):
+        super(PCADecomposition, self).fit(X=X, y=y, **kwargs)
         self.pca_transformer.fit(X)
+        self.pca_components_ = self.pca_transformer.steps[1][1].components_
```

Comment:
Excellent, thank you for adding this to a learned `pca_components_` attribute, I think this is the exact correct place to store this information. 

In the future, I may update this to `self.pca_transformer.named_steps['pca'].components_` -- but only for understandability to new contributors; this is perfectly fine right now!

Reply:
I did not know about "named_steps", I can fix it at night.

Reply:
Don't worry about it! Using steps is perfectly acceptable; I'd like to go ahead an merge this now if that's ok with you, then I can quickly add the visual tests, and we can get back on track with #430!